README.md

```diff
@@ -25,13 +24,12 @@ It includes an optional label,error message,invalid,autovalidate and required at
 
 If the `auto-validate` attribute is set, element validates when the user not select the state from dropdown menu.
 
-
 ## Install
 
 Install the component using [Bower](http://bower.io/):
 
 ```sh
-$ bower install — save biztek/usa-state-list
+$ bower install -- save biztek/usa-state-list
```

Comment:
@anushagadepalli 
Please remove the space before `save`. It should say `--save`

Also, please remove the `$` sign.